src/test/java/com/commercetools/sync/inventories/InventorySyncTest.java

```diff
@@ -175,14 +175,33 @@ void sync_WithEnsuredChannels_ShouldCreateEntriesWithUnknownChannels() {
         assertThat(errorCallBackExceptions).hasSize(0);
     }
 
+    @Test
+    void sync_WithErrorOnCreatingInventory_ShouldCallErrorCallbacl() {
+        final InventoryEntryDraft draftWithNewChannel = InventoryEntryDraft.of(SKU_3, QUANTITY_1, DATE_1, RESTOCKABLE_1,
+            Channel.referenceOfId(KEY_3));
+        final InventorySyncOptions options = getInventorySyncOptions(30, true);
+        final InventoryService inventoryService = getMockInventoryService(existingInventories,
+            Optional.empty(),  mock(InventoryEntry.class));
+        final ChannelService channelService = getMockChannelService(getMockSupplyChannel(REF_2, KEY_2));
+
+        InventorySync inventorySync = new InventorySync(options, inventoryService, channelService,
+            mock(TypeService.class));
+        final InventorySyncStatistics stats = inventorySync.sync(singletonList(draftWithNewChannel))
+            .toCompletableFuture()
+            .join();
+
+        assertThat(stats).hasValues(1, 0, 0, 1);
```

Comment:
How do you know from this test that error callback has been called?

Reply:
In this case no error callback is called, so I renamed the method  -> .20bdf678